tests/test_inclined_exponential.py

```diff
@@ -30,155 +38,319 @@
+                    verbose=True)
+
+    # Now do Sersic-only tests
+    for (flux, sersic_n, inc_angle, scale_radius, scale_height,
+         trunc_factor, pos_angle) in inclined_sersic_regression_test_parameters:
+
+        image_filename = ("galaxy_" + sersic_n + "_" + inc_angle + "_" + scale_radius +
+                          "_" + scale_height + "_" + trunc_factor + "_" + pos_angle + ".fits")
+        print("Comparing " + mode + " against " + image_filename + "...")
 
         # Get float values for the details
-        inc_angle=float(inc_angle)
-        scale_radius=float(scale_radius)
-        scale_height=float(scale_height)
-        pos_angle=float(pos_angle)
+        flux = float(flux)
+        sersic_n = float(sersic_n)
+        inc_angle = float(inc_angle)
+        scale_radius = float(scale_radius)
+        scale_height = float(scale_height)
+        trunc_factor = float(trunc_factor)
+        pos_angle = float(pos_angle)
+
+        image = galsim.fits.read(image_filename, image_dir)
+        image *= flux / image.array.sum()
+        nx, ny = np.shape(image.array)
 
         # Now make a test image
-        test_profile = galsim.InclinedExponential(inc_angle*galsim.radians, scale_radius,
-                                                  scale_height)
-        check_basic(test_profile, "InclinedExponential")
+        test_profile = get_prof("InclinedSersic", n=sersic_n, scale_radius=scale_radius, scale_height=scale_height,
+                                inclination=inc_angle * galsim.radians, trunc=trunc_factor * scale_radius, flux=flux)
+        check_basic(test_profile, mode)
 
         # Rotate it by the position angle
-        test_profile = test_profile.rotate(pos_angle*galsim.radians)
+        test_profile = test_profile.rotate(pos_angle * galsim.radians)
 
         # Draw it onto an image
-        test_image = galsim.Image(image_nx,image_ny,scale=1.0)
-        test_profile.drawImage(test_image,offset=(0.5,0.5)) # Offset to match Lance's
+        test_image = galsim.Image(nx, ny, scale=1.0)
+        test_profile.drawImage(test_image, offset=(0.5, 0.5)) # Offset to match Lance's
+
+        # Save if desired
+        if save_profiles:
+            test_image_filename = image_filename.replace(".fits", "_" + mode + ".fits")
+            test_image.write(test_image_filename, image_dir, clobber=True)
 
         # Compare to the example - Due to the different fourier transforms used, some offset is
         # expected, so we just compare in the core to two decimal places
 
-        image_core = image.array[image_ny//2-2:image_ny//2+3, image_nx//2-2:image_nx//2+3]
-        test_image_core = test_image.array[image_ny//2-2:image_ny//2+3, image_nx//2-2:image_nx//2+3]
-
-        ratio_core = image_core / test_image_core
-
-        # galsim.fits.write(test_image,"test_"+image_filename,image_dir)
+        image_core = image.array[nx // 2 - 2:nx // 2 + 3, ny // 2 - 2:ny // 2 + 3]
```

Comment:
I know this is PEP8 compliant, but this would be much more readable with spaces around the colons, and not around the //, +, -.